spec/requests/api/projects_spec.rb

```diff
@@ -117,59 +117,34 @@
     end
   end
 
-  describe 'POST /projects' do
-    context 'maximum number of projects reached' do
-      before do
-        (1..user2.projects_limit).each do |project|
-          post api('/projects', user2), name: "foo#{project}"
-        end
-      end
-
-      it 'should not create new project' do
+  describe "POST /projects" do
+    context "maximum number of projects reached" do
+      it 'should not create new project and respond with 403' do
+        User.any_instance.stub(:projects_limit_left).and_return(0)
         expect {
-          post api('/projects', user2), name: 'foo'
+          post api('/projects', user2), name: "foo"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.